src/components/dashboard/forms/items/Campaign/NewCampaignHoc.js

```diff
@@ -47,6 +47,7 @@ export default function NewCampaignHoc(Decorated) {
 			this.handleChange('temp', newTemp)
 			this.onSave()
 			actions.resetNewItem('Campaign')
+			actions.checkNetworkCongestion()
```

Comment:
I'm not sure it has to be at save. It will be overwritten by success toast.
It is better to make this check on the preview page